pkg/aws/eni/node_test.go

```diff
@@ -0,0 +1,42 @@
+// Copyright 2019-2020 Authors of Cilium
```

Comment:
_I am not a layer_, but shouldn't copyright start in 2020?

Reply:
absolutely! 👍